rfc6962-bis.xml

```diff
@@ -1177,6 +1177,9 @@ entry specified by <spanx style="verb">start</spanx>.
           </t>
         </list>
       </t>
+	  <t>
+        Additionally, a TLS server which supports presenting SCTs using an OCSP response MAY provide it when the TLS client included the <spanx style="verb">status_request_v2</spanx> extension (<xref target='RFC6961'/>) in the (extended) <spanx style="verb">ClientHello</spanx>, but only in addition to the <spanx style="verb">status_request</spanx> extension mentioned above.
```

Comment:
I'd forgotten that both status_request and status_request_v2 use the same CertificateStatus message.  But despite that, I think that 6962-bis should neither require status_request_v2 to be supported nor require status_request to never be deprecated.